exercises/prime-factors/prime_factors_test.exs

```diff
@@ -62,4 +62,16 @@ defmodule PrimeFactorsTest do
   test "93819012551" do
     assert PrimeFactors.factors_for(93819012551) == [11, 9539, 894119]
   end
+
+  @tag :optional
+  @tag timeout: 120000
+  test "performance optimizations" do
+    %{minute: original_minutes, second: original_seconds} = DateTime.utc_now
+    PrimeFactors.factors_for(10000000055) == [5, 2000000011]
```

Comment:
Good point - I'm not sure we care about the result here, but it couldn't hurt to add an assertion there. Or maybe split it into two tests - one to get the value, and the second to ensure that it finishes in a reasonable amount of time?